.github/workflows/edge-py-release.yml

```
@@ -0,0 +1,196 @@
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    needs:
+      - merge-artifacts
+    # Only run if triggered by a tag push
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - uses: actions/download-artifact@v4
+        with:
+          name: wheels
+      - uses: actions/setup-python@v2
```

actionlint (1.7.9)

the runner of "actions/setup-python@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)